report/finished/Edward_C_evalulation_Effector_Arm.tex

```diff
@@ -0,0 +1,19 @@
+
+\subsection{Robotic arm}
+In the beginning many different arm choices was discussed. One solution was using a simpler forklift design that just moves up and forward in a rigid motion without rotation.
+This solution would suffice. However, we decided to go with a arm design with links connected with motors.
+This is a more complex solution but opens up translation and rotation in 3D-space.
```

Comment:
"opens up **for** translation"  
I don't think it is wrong but maybe it gets clearer?